build/dependenciesTest.props

```diff
@@ -3,6 +3,7 @@
     <BannedApiAnalyzersVersion>3.3.4</BannedApiAnalyzersVersion>
     <DotNetCoreAppRuntimeVersion>2.1.30</DotNetCoreAppRuntimeVersion>
     <MicrosoftAzureKeyVaultCryptographyVersion>2.0.5</MicrosoftAzureKeyVaultCryptographyVersion>
+    <MicrosoftExtensionsTimeProviderTestingVersion>8.8.0</MicrosoftExtensionsTimeProviderTestingVersion>
```

Comment:
Can we use latest?

Reply:
Also this is unused, pending the other discussion about using the testing package.

Reply:
We should consider removing this until we need it.

Reply:
Removed, I will follow-up with .NET team to understand the issue with this odd warning. Thanks Ignacio & Keegan!